scripts/load-docker-compose.sh

```diff
@@ -13,6 +13,7 @@
 #	          -M herd-registry-mellerud.xlsx
 #
 
+# shellcheck disable=SC2266
```

Comment:
and what are we disabling here?

Reply:
We pipe to a `test`-like expression (`[[ .. ]]`) which is typically not correct.